.github/workflows/deploy.yml

```diff
@@ -4,23 +4,22 @@ on:
   release:
     types: [published]
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    timeout-minutes: 5
-    
+    timeout-minutes: 10
+
     steps:
     - name: Checkout repo
-      uses: actions/checkout@v2.1.0
+      uses: actions/checkout@v2
       with:
         submodules: true
-    
+
     - name: Setup Node.js environment
-      uses: actions/setup-node@v1.4.2
+      uses: actions/setup-node@v2
       with:
-        node-version: '12.x'
-    
+        node-version: 14
```

Comment:
Will this pull in the latest minor version or do we need `14.x` here?

Reply:
The instructions at https://github.com/actions/setup-node now say just give the number instead of `.x`